src/Tribe/Admin/Notices.php

```diff
@@ -237,4 +238,44 @@ public function maybe_display_plus_commerce_notice() {
 			tribe_notice( "event-tickets-plus-missing-{$provider}-support", $message, 'dismiss=1&type=warning' );
 		}
 	}
+
+	/**
+	 * Display notices for unsupported currencies.
+	 *
+	 * @return void
+	 */
+	public function maybe_display_unsupported_currency_notice() {
+		// Get currencies and notice texts.
+		$unsupported_currency = [
+			'HRK' => [
+				'heading' => __( 'Tickets Commerce is now selling with Euro', 'event-tickets' ),
+				'message' => __( 'From the 1st of January 2023, the euro became the official currency for Croatia. We have removed the Croatian kuna from our currency settings and updated your settings to start selling with Euro.', 'event-tickets' ),
+				'updated' => 'EUR'
+			],
+		];
+
+		// Get currency code option
+		$currency = tribe_get_option( 'tickets-commerce-currency-code' );
+
+		if ( array_key_exists( $currency, $unsupported_currency ) ) {
+
+			$message = sprintf(
+				'<h3>%1$s</h3><p>%2$s</p>',
+				esc_html( $unsupported_currency[$currency]['heading'] ),
+				esc_html( $unsupported_currency[$currency]['message'] )
+			);
+
+			tribe_notice(
+				__FUNCTION__,
+				$message,
+				[
+					'dismiss' => true,
+					'type'    => 'warning',
+				]
+			);
+
+			// Update currency to new option
+			tribe_update_option( 'tickets-commerce-currency-code', $unsupported_currency[$currency]['updated'] );
```

Comment:
I'm not sure if we should include this update logic within the notification. In my opinion, the logic around unsupported currencies and the update of the option should live in the [currency class](https://github.com/the-events-calendar/event-tickets/blob/master/src/Tickets/Commerce/Utils/Currency.php), and then we consume that data from the notice function.

That way we keep organization and store all the currency-related functionality in the currency class, and then keep this notification class only with the notice functionality. Feel free to reach out if you have doubts about this.

```suggestion
			tribe_update_option( 'tickets-commerce-currency-code', $unsupported_currency[ $currency ]['updated'] );
```

Reply:
@juanfra So true. Thanks for this! I'll refactor this into the Currency class.